### ISSUE TITLE
Support Tumblr Alt Text

### ISSUE THREAD

**Issue by alexamacbett** [mvdicarlo/postybirb#239]
I'm not actually certain if this is possible, since I couldn't tell from Tumblr's documentation whether it supports alt text on images via the api, but assuming that it does it would be great to support alt text for image posts/images in posts, since it's not possible to edit alt text onto already posted images on Tumblr meaning that users who either have policies of only posting images with alt-text or who only post images that should be alt-texted anyway due to their content (such as ads containing large amounts of text) cannot use the tumblr integration of postybirb currently.

**Comment by mvdicarlo** [comment]
This feature is coming to V4.
